.github/actions/spell-check/whitelist.txt

```diff
@@ -0,0 +1,3240 @@
+aaaaa
+aaaaaa
+aaaaaaaaaaaaaaaaaaa
+AAAAAABBBBBBCCC
+AAAAABBBBBBBCCC
+AAD
+ABANDONFONT
+abcd
+abcde
+abcdef
+ABCDEFG
+ABCDEFGH
+ABCDEFGHIJ
+abcdefghijk
+abcdefghijklmnop
+ABCDEFGHIJKLMNOPQRST
+abcdefghijklmnopqrstuvwxyz
+ABCG
+abe
+abec
+abf
+abi
+acb
+accd
+ACCEPTFILES
+ACCESSDENIED
+acec
+acf
+acidev
+ACIOSS
+ACover
+actctx
+ACTCTXW
+activatable
+ACTIVEBORDER
+ACTIVECAPTION
+ADDALIAS
+ADDREF
+addressof
+ADDSTRING
+ADDTOOL
+AEB
+aeed
+aef
+AEnd
+afae
+afceeeaa
+AFE
+AFEF
+AFew
+AFill
+AFX
+AHelper
+ahz
+AImpl
+AInplace
+akb
+ALIGNRIGHT
+alloc
+allocing
+alpc
+ALTERNATENAME
+ALTF
+ALTNUMPAD
+ALWAYSTIP
+amd
+ansd
+ansicode
+ansicpg
+ANSISYS
+ANSISYSRC
+ANSISYSSC
+antialias
+antialiasing
+ANull
+Anup
+anycpu
+AOn
+APARTMENTTHREADED
+APCs
+api
+APIENTRY
+apiset
+apos
+appconsult
+appdata
+APPICON
+appium
+applet
+appletname
+applicationmodel
+APPLMODAL
+APPLOGIC
+appmodel
+apps
+APPWINDOW
+appx
+appxbundle
+Appxmandifest
```

Comment:
This is a typo. `Appxmandifest` should be `appxmanifest`.

Reply:
f8965f6